Add automatic generation of conda binary packages for Windows, macOS and Linux in robotology channel (#652)
robotology/robotology-superbuild: doc/conda-forge.md

~~~
conda create -n robotologyenv
~~~

Once you created the `robotologyenv` environment, you can "activate" it for the current terminal (i.e. make sure that the installed packages can be found) by the command:
~~~
conda activate robotologyenv
~~~

**IMPORTANT: if you open a new terminal, you need to manually activate the environment also there.**

### Install robotology packages

Once you are in an activated environment, you can install robotology packages by just running the command:
~~~
conda install -c robotology <packagename>
~~~

The list of available packages is available at https://anaconda.org/robotology/repo . 

For example, if you want to install yarp and icub-main, you simple need to install:
~~~
conda install -c robotology yarp icub-main
~~~

While if you want to simulate the iCub in Gazebo, you can install `icub-models` and `gazebo-yarp-plugins`:
~~~
conda install -c robotology gazebo-yarp-plugins icub-models
~~~

If you want to develop some C++ code on the top of this libraries, it is recommended to also install the necessary compiler and development tools directly in the same environment:
~~~

> conda installl compilers cmake pkg-config make ninja

Reviewer: Typo installl

For this reason, to use the robotology conda packages it is suggested to first create a conda environment, and then install in it all the packages you want to use. To create a new environment called `robotologyenv`, execute the following command:
~~~
conda create -n robotologyenv
~~~

Once you created the `robotologyenv` environment, you can "activate" it for the current terminal (i.e. make sure that the installed packages can be found) by the command:
~~~
conda activate robotologyenv
~~~

**IMPORTANT: if you open a new terminal, you need to manually activate the environment also there.**

### Install robotology packages

Once you are in an activated environment, you can install robotology packages by just running the command:
~~~
conda install -c robotology <packagename>
~~~

The list of available packages is available at https://anaconda.org/robotology/repo . 

For example, if you want to install yarp and icub-main, you simple need to install:
~~~
conda install -c robotology yarp icub-main
~~~

> While if you want to simulate the iCub in Gazebo, you can install `icub-models` and `gazebo-yarp-plugins`:

Reviewer 2: ```suggestion
In addition, if you want to simulate the iCub in Gazebo, you should also install `icub-models` and `gazebo-yarp-plugins`:
```

~~~
conda install -c robotology gazebo-yarp-plugins icub-models
~~~

> If you want to develop some C++ code on the top of this libraries, it is recommended to also install the necessary compiler and development tools directly in the same environment:

Reviewer 2: ```suggestion
If you want to develop some C++ code on the top of these libraries, it is recommended to also install the necessary compiler and development tools directly in the same environment:
```
